elements/classpower.lua

```diff
@@ -243,26 +252,27 @@ local function Enable(self, unit)
 		element.ClassPowerEnable = ClassPowerEnable
 		element.ClassPowerDisable = ClassPowerDisable
 
-		local isChildrenTextures
+		local isStatusBar
 		for i = 1, #element do
 			local icon = element[i]
```

Comment:
Change this variable (and all the following) to `bar`? I personally like `pill`, but w/e.